Fix clean network example performance cf

cforgaci opened the pull request:
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Optimization
- [x] Documentation Update

## Description
The CRAN submission flagged the example in `clean_network()` as taking too long in “Examples with CPU (user + system) or elapsed time > 5s” or "> 10s". See logs below. With this PR, the example is only run in intercative mode. I also updated the version to 0.1.1.

```
Flavor: r-devel-windows-x86_64
Check: CRAN incoming feasibility, Result: NA
  Maintainer: 'Claudiu Forgaci <EMAIL_REMOVED>'
  
  New submission
  
  Possibly misspelled words in DESCRIPTION:
    Forgaci (16:60)

Flavor: r-devel-windows-x86_64
Check: examples, Result: NA
  Examples with CPU (user + system) or elapsed time > 10s
                 user system elapsed
  clean_network 13.84   1.31   15.42

Flavor: r-devel-windows-x86_64
Check: Overall checktime, Result: NOTE
  Overall checktime 13 min > 10 min

Flavor: r-devel-linux-x86_64-debian-gcc
Check: CRAN incoming feasibility, Result: NOTE
  Maintainer: 'Claudiu Forgaci <EMAIL_REMOVED>'
  
  New submission
  
  Possibly misspelled words in DESCRIPTION:
    Forgaci (16:60)

Flavor: r-devel-linux-x86_64-debian-gcc
Check: examples, Result: NOTE
  Examples with CPU (user + system) or elapsed time > 5s
                 user system elapsed
  clean_network 6.451  0.264       7
```

## Related Issues

- Related Issue #
- Closes #

## Added/updated tests?
_We encourage you to keep the code coverage percentage at 75% and above._

- [ ] Yes
- [x] No, and this is why: no need for tests
- [ ] I need help with writing tests

## Added entry in changelog?
_For user-facing changes, add a line describing the changes in [NEWS.md](/NEWS.md)_

- [x] Yes

cforgaci commented:
> Thanks, was `valley.R` accidentally left out from the commit?

No, that example was updated before in `valley.R`; only the documentation was not.

fnattino reviewed (commented):
Should the `cran-comments.md` also be uptaded indicating the change?

cforgaci commented:
> The CRAN submission flagged the example in `clean_network()` as taking too long in “Examples with CPU (user + system) or elapsed time > 5s” or "> 10s". See logs below. With this PR, the example is only run in intercative mode. I also updated the version to 0.1.1.

@fnattino, thanks, I missed that. Updated now.
